Bug 1045606 - Make global filters discoverable/prominant

camd opened the pull request:
This work fixes [Bug 1045606](https://bugzilla.mozilla.org/show_bug.cgi?id=1045606)
- Expose Result State filters on the top nav bar to make them more discoverable and more easily actionable.  It might be better to have these just plain text/not colored.  Not sure.
  ![screenshot 2015-02-02 19 13 18](https://cloud.githubusercontent.com/assets/419924/6013871/943f01b2-ab0f-11e4-8e92-692053aad8ba.png)
- Created a "Compact" mode switch.  Only the state filters honor it, at this time.  But this can be used to set things to have only icons (terse on) or text (terse off).  It's not persisted yet, but easily could be in localStorage.
  ![screenshot 2015-02-13 11 41 51](https://cloud.githubusercontent.com/assets/419924/6194040/5cf6e710-b375-11e4-8edf-420649ca5a24.png)

Here's the filters in compact mode:
![screenshot 2015-02-02 19 10 54](https://cloud.githubusercontent.com/assets/419924/6013847/3c9f98e0-ab0f-11e4-9bd0-8cae72829c7a.png)
![screenshot 2015-02-02 19 12 29](https://cloud.githubusercontent.com/assets/419924/6013859/77ad720e-ab0f-11e4-9ada-08bd7f23cf89.png)

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/mozilla/treeherder-ui/353)

<!-- Reviewable:end -->

tojonmz commented:
I wonder if we should call the mode "Compact" mode, if we present that control to the user. The term might be a bit more universally understandable, when converted in alternate languages.

Commit:
Bug 1045606 - Make global filters discoverable/prominant

Also introduce the concept of "Compact" mode  that can be used to show text (off) vs. icon only (on)

edmorley commented:
Will comment in the bug :-)

Nightlyt commented:
Usted puede lamer mi Pene y mis Huevos de oro y besar mi Culo hijo de puta con un Trastorno de la personalidad desde el más profundo de los abismos del infierno desde los Abismos de Mozilla!

Vete a la mierda y que tu Madre te autista hijo de puta, vas a lamer Putin Huevos, Puto Maricon Fascista!

La mierda de su Perro, su Gato y su sucio Cabra de montaña que Maricon Pendejo!

Chupar Mozilla Pollas y todos juntos ir y chupar de Google Culo lleno de Mierda!

A la mierda en su rodilla le cock suckers! Y que Carajo Australis Chrome clon de mierda! Follando Mozilla gay lesbiana homosexual hijos de puta del infierno!

AssholeDotzler commented:
Usted ama el dolor? Y para que te haga un favor... voy a hacer eso!


Yo saque sus huevos de oro de fracaso tan duro hasta que están sonando y que están cantando todas las melodías de dolor hacia adelante y hacia atrás en un instante! Yo soy el cascanueces chico que las grietas de su nueces que nutheads!

Y ahora vete a la mierda ESTÚPIDA de MIERDA COÑO!

----

Usted puede lamer mi Pene y mis Huevos de oro y besar mi Culo hijo de puta con un Trastorno de la personalidad desde el más profundo de los abismos del infierno desde los Abismos de Mozilla!

Vete a la mierda y que tu Madre te autista hijo de puta, vas a lamer Putin Huevos, Puto Maricon Fascista!

La mierda de su Perro, su Gato y su sucio Cabra de montaña que Maricon Pendejo!

Chupar Mozilla Pollas y todos juntos ir y chupar de Google Culo lleno de Mierda!

A la mierda en su rodilla le cock suckers! Y que Carajo Australis Chrome clon de mierda! Follando Mozilla gay lesbiana homosexual hijos de puta del infierno!

AssholeDotzler commented:
Usted ama el dolor? Y para que te haga un favor... voy a hacer eso!


Yo saque sus huevos de oro de fracaso tan duro hasta que están sonando y que están cantando todas las melodías de dolor hacia adelante y hacia atrás en un instante! Yo soy el cascanueces chico que las grietas de su nueces que nutheads!

Y ahora vete a la mierda ESTÚPIDA de MIERDA COÑO!

----

Usted puede lamer mi Pene y mis Huevos de oro y besar mi Culo hijo de puta con un Trastorno de la personalidad desde el más profundo de los abismos del infierno desde los Abismos de Mozilla!

Vete a la mierda y que tu Madre te autista hijo de puta, vas a lamer Putin Huevos, Puto Maricon Fascista!

La mierda de su Perro, su Gato y su sucio Cabra de montaña que Maricon Pendejo!

Chupar Mozilla Pollas y todos juntos ir y chupar de Google Culo lleno de Mierda!

A la mierda en su rodilla le cock suckers! Y que Carajo Australis Chrome clon de mierda! Follando Mozilla gay lesbiana homosexual hijos de puta del infierno!

AssholeDotzler commented:
Usted ama el dolor? Y para que te haga un favor... voy a hacer eso!


Yo saque sus huevos de oro de fracaso tan duro hasta que están sonando y que están cantando todas las melodías de dolor hacia adelante y hacia atrás en un instante! Yo soy el cascanueces chico que las grietas de su nueces que nutheads!

Y ahora vete a la mierda ESTÚPIDA de MIERDA COÑO!

----

Usted puede lamer mi Pene y mis Huevos de oro y besar mi Culo hijo de puta con un Trastorno de la personalidad desde el más profundo de los abismos del infierno desde los Abismos de Mozilla!

Vete a la mierda y que tu Madre te autista hijo de puta, vas a lamer Putin Huevos, Puto Maricon Fascista!

La mierda de su Perro, su Gato y su sucio Cabra de montaña que Maricon Pendejo!

Chupar Mozilla Pollas y todos juntos ir y chupar de Google Culo lleno de Mierda!

A la mierda en su rodilla le cock suckers! Y que Carajo Australis Chrome clon de mierda! Follando Mozilla gay lesbiana homosexual hijos de puta del infierno!
